src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -4,14 +4,9 @@
 
 import java.util.Arrays;
 
-/**
- * Bogna 'bognix' Knychala
- */
 public class MobileAdsDataProvider {
 
-  private MobileAdsDataProvider() {
-
-  }
+  private MobileAdsDataProvider() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)